setup.py

```diff
@@ -83,11 +83,22 @@ def run(self):
 
 # build_py will produce plat_name = 'any'. Patch the bdist_wheel to change the
 # platform tag because the C extensions are platform dependent.
+# For setuptools<70
 from wheel.bdist_wheel import bdist_wheel
-initialize_options = bdist_wheel.initialize_options
+initialize_options_1 = bdist_wheel.initialize_options
 def initialize_with_default_plat_name(self):
-    initialize_options(self)
+    initialize_options_1(self)
     self.plat_name = get_platform()
+    self.plat_name_supplied = True
+bdist_wheel.initialize_options = initialize_with_default_plat_name
+
+# For setuptools>=70
+from setuptools.command.bdist_wheel import bdist_wheel
```

Comment:
Will this be okay if you install from source (eg `pip install --no-binary pyscf pyscf`) but have setuptools <70.1.0?

Not sure whether it is better to require setuptools >=70.1, check the version number of setuptools, or check whether setuptools.command.bdist_wheel exists...

Reply:
Actually, I'm not sure which version of setuptools causes the problem. Just a general fix so that all setuptools>61 work

Reply:
According to setuptools' webpage, bdist_wheel was [added](https://setuptools.pypa.io/en/latest/history.html#v70-1-0) in the latest version, 70.1.0. Given the timing of the Github Actions build failure, this version seems to be the culprit!